db/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2023-08-19 23:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
```

Comment:
use " instaed of ', use Black

Reply:
how can I change ' for " in the automatically created file?  Should I change migration file every time I create them by 
"python manage.py makemigrations db"? or how should it work? I really don't get it.